source/sass/main.scss

```diff
@@ -26,15 +26,32 @@ img {
   max-width: 100%;
 }
 
-ul {
```

Comment:
We shouldn't need this anyways. List in markdown should be styled as a list when it gets rendered. 😃